grammar/haxe.bnf

```diff
@@ -383,7 +383,7 @@ blockStatement ::= '{' statementList? '}' {mixin="com.intellij.plugins.haxe.lang
 private statementList ::= (statement ';'?)+ {recoverWhile="statement_recovery"}
 private statement_recovery ::= !('case' | 'default' | '}')
 
-private statement ::= blockStatement | notBlockStatement
+private statement ::= macroClassList? (blockStatement | notBlockStatement)
```

Comment:
2 failed, one because it wasn't updated, other is HaxeCompilerError/Warning test case